test/Microsoft.AspNet.Mvc.Formatters.Json.Test/JsonInputFormatterTest.cs

```diff
@@ -38,7 +38,11 @@ public void CanRead_ReturnsTrueForAnySupportedContentType(string requestContentT
             var contentBytes = Encoding.UTF8.GetBytes("content");
 
             var httpContext = GetHttpContext(contentBytes, contentType: requestContentType);
-            var formatterContext = new InputFormatterContext(httpContext, new ModelStateDictionary(), typeof(string));
```

Comment:
Good catch. I meant to vary the new `FromBodyAndRequiredOnInnerValueTypeProperty_NotBound_JsonFormatterSuccessful()` until I got deserialization until it failed lower down but didn't go back and finish after rebasing. Will add another test or two here or in `BodyValidationIntegrationTests`...